website/sidebars.json

```diff
@@ -47,7 +47,12 @@
       "attached-class",
       "intersection-types",
       "generics",
-      "non-forcing-constants"
+      "non-forcing-constants",
+      "strong"
+    ],
+    "Editor Features": [
+      "sig-suggestion",
+      "highlight-untyped"
```

Comment:
I want to keep expanding this section with more editor-specific docs. Right now the VS Code docs are terribly lacking. There are plenty more editor-specific docs that should exist, but I have always hesitated to write them because mentally I figured that I needed to rewrite all the VS Code docs all at once. I don't think that's a particularly productive mentality, and we can take a more incremental approach.

For example another page I want is to document the code actions, and I want to explain Sorbet's custom extensions, and I want a doc for what the "Typechecking..." etc. statuses mean, etc.